test/client/RepositoryList.test.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { createRenderer } from 'react-addons-test-utils'
+import { expect } from 'chai'
+
+import RepositoryList from '../../client/components/RepositoryList.jsx'
+
+describe('RepositoryList', () => {
+
+  const fixtures = {
+    repositories: {
+      'zappr/aaa': {
+        full_name: 'zappr/aaa'
+      },
+      'zappr/ccc': {
+        full_name: 'zappr/ccc'
+      },
+      'zappr/bbb': {
+        full_name: 'zappr/bbb'
```

Comment:
Please also include some uppercase repo names. I know they don't exist in the API, but if they suddenly appear we want consistent sorting regardless.